### DISCUSSION TITLE
`npx tailwindcss` doesn't seem to detect my template files but `npc tailwindcss-cli` works totally fine

### DISCUSSION THREAD

**Discussion by allreddy** [tailwindlabs/tailwindcss#11776] [Help]
Been trying to set up Tailwind with my Django project for the last few hours, checking, double checking and triple checking the configs for the template directories to no avail.

``` 
$ npx tailwindcss -i ./static/src/input.css -o ./static/src/output.css --watch --diagnostics    

Rebuilding...

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration

Done in 76ms.
```

Found this StackOverflow [thread](https://stackoverflow.com/a/72931081/3516614) pointing me in the direction of using `tailwindcss-cli` with a warning that it hasn't been updated since 2020.

```
$ npx tailwindcss-cli -i ./static/src/input.css -o ./static/src/output.css --watch --diagnostics
Done in 3605ms.

```

And it seems to work fine. What's the guidance here?

Is it advisable to continue with `tailwindcss-cli`? If not, what's the issue with `tailwindcss` failing to detect the template files?

Any assistance here would be much appreciated.

**Comment by wongjn** [comment]
Consider checking that [the `content` file globs cover source code](https://tailwindcss.com/docs/content-configuration) that use Tailwind classes.

**Reply by allreddy** [reply]
I mean it clearly does because `tailwindcss-cli` is recognizing the utility classes I've added in my html files found at `./templates/**/*.html` ,
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
      "./templates/**/*.html",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

```
